ddtrace/sampler.py

```diff
@@ -44,8 +45,8 @@
 
 log = get_logger(__name__)
 
-MAX_TRACE_ID = 2 ** 64
-
+# TODO: deprecate and remove MAX_TRACE_ID, it is no longer used.
```

Comment:
```suggestion
```

Reply:
also we're subtly changing this value now (from 2^64 to 2^64-1)

Reply:
Yup I think 2**64 is incorrect. MAX TRACE ID should be equal to max uint64